### PR TITLE
Add sites_enabled_path and streams_enabled_path param

### PR DESCRIPTION
**Pull request by feltra**
Allow user to change path for sites-enabled and streams-enabled keeping it within conf dir.
related to: #1234 and #1302

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -52,6 +52,7 @@
   $sites_available_owner                                     = $nginx::params::sites_available_owner,
   $sites_available_group                                     = $nginx::params::sites_available_group,
   $sites_available_mode                                      = $nginx::params::sites_available_mode,
+  $sites_enabled_path                                        = $nginx::params::sites_enabled_path,
```
Comment (bastelfreak) [inline manifests/init.pp]: Please add a Datatype like `String[1]` to this

File: manifests/init.pp [hunk 2]
```diff
@@ -97,6 +98,7 @@
   $log_format                                                = {},
   Boolean $mail                                              = false,
   Boolean $stream                                            = false,
+  $streams_enabled_path                                      = $nginx::params::sites_enabled_path,
```
Comment (bastelfreak) [inline manifests/init.pp]: Please add a Datatype like `String[1]` to this

**Comment by bastelfreak** [comment]
Ping @feltra :)

**Comment by TheMeier** [comment]
Closing due to inactivity. Please re-open if this is still relevant.

**Pull request closed without merging**